R/constants.R

```diff
@@ -70,6 +70,7 @@ PRICERATIO_GRASS_ALFALFA <- 0.7
 # NUMBERS OF DIGITS FOR MODEL INPUT DATA
 aglu.DIGITS_CALPRICE <- 4 # prices and costs
 aglu.DIGITS_CALOUTPUT <- 7 # production
+aglu.DIGITS_INCELAS <- 4 # food demand income elasticity
```

Comment:
This is probably a bigger picture question than just this PR, but do we really need all of these different rounding levels?